web/src/billing/billing.ts

```diff
@@ -377,10 +376,8 @@ export function initialize(): void {
             url: `/json${billing_base_url}/billing/plan`,
             data,
             success() {
-                window.location.replace(
-                    `${billing_base_url}/billing/?success_message=` +
-                        encodeURIComponent("Billing frequency has been updated."),
-                );
+                sessionStorage.setItem("success_message", "Billing frequency has been updated.");
```

Comment:
I don't love this approach; we want something that is clearly tied to this specific request. Could we just pass the parameter via the URL `#` component, and have the page drop that from the URL after processing it, just like the Zulip web app does for various things?

Reply:
That make sense to do given that it doesn't matter if we show the `success_message` in the URL temporarily. Thanks! Updated the PR with that change.